README.md

```diff
@@ -1,127 +1,71 @@
-# 兄弟们，哥们早从Android脱坑了，入了后台的坑，该项目早已停止维护，要用于生产环境的请慎重！
+# VerificationView
 
-# 一、添加依赖
+A HMOS library which provides Verification code features.
 
-1. **Android Studio**：在项目的根目录下build.gradle里的allprojects下添加如下引用：
+## Source
 
-```groovy
-   allprojects {
-   	repositories {
-   		...
-   		maven { url 'https://jitpack.io' }
-   	}
-   }
-```
-
-   然后在app目录下 build.gradle 下添加依赖
-
-```groovy
-   dependencies {
-   	   compile 'com.github.yongming9011:VerificationCodeView:v1.0'
-   }
-```
-
-   ​
-
-2. **Eclipse**：下载项目，将library里面的VerificationCodeView.java跟res/values/attrs.xml文件拷贝进项目。
-
-# 二、用法
-
-- 在布局中集成，注意需要在布局的根节点添加命名`xmlns:zhangym="http://schemas.android.com/apk/res-auto"` 方可使用自定义属性。
-
-  ```xml
-   <com.zhangym.customview.VerificationCodeView
-          android:id="@+id/verificationCodeView"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_centerInParent="true"
-          android:padding="20dp"
-          zhangym:interferenceCirclesCount="40"
-          zhangym:interferenceCirclesRadius="5"
-          zhangym:interferenceLinesCount="10"
-          zhangym:isShowInterferenceCircles="true"
-          zhangym:isShowInterferenceLines="true"
-          zhangym:isTextBold="true"
-          zhangym:textSize="50sp"
-          zhangym:textSkewX="0"
-          zhangym:verificationText="9D27" />
-  ```
-
-
-- 对应属性说明如下：
+Inspired by [yongming9011/VerificationCodeView](https://github.com/yongming9011/VerificationCodeView)
```

Comment:
Base library has a github release. Add the version number of the baselib that was used for reference.